composer.json

```diff
@@ -12,8 +12,9 @@
 		}
 	],
 	"require": {
-		"php": "^8.1 || ^8.2",
-		"typo3/cms-core": "^11.5 || ^12.0 || dev-main"
+		"php": "^8.1 || ^8.2 || ^8.3",
```

Comment:
doesn't ^8.1 alone include 8.3.x?

Reply:
That syntax is not from me, I just adapted it. I assumed, that is intended to use `^8.1 || ^8.2`. Some people make this to stress that it is tested for 8.1 and 8.2 but will also work with 8.3 presumably.

Reply:
ok, why not